crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -49,6 +49,9 @@ maplit = { version = "1", optional = true }
 mockall = { version = "0.10.2", optional = true }
 
 [dev-dependencies]
+# include self with test_utils feature, to allow integration tests to run properly
+kitsune_p2p = { path = ".", features = ["test_utils"]}
```

Comment:
I'd never seen this before... will have to try to remember next time I run into this problem.

Reply:
It's new in resolver 2 (default in edition 2021)